app/javascript/packs/sentry.js

```diff
@@ -0,0 +1,16 @@
+import * as Sentry from "@sentry/browser";
+import { Integrations } from "@sentry/tracing";
+
+if (typeof Sentry !== 'undefined') {
+  Sentry.init({
+      dsn: "https://8e3b0c54e017453aa0852630e8d61231@sentry.ds.agency/4",
```

Comment:
is it some kind of token?
I don't see sentry credentials usage

Reply:
Yes. you want to move it to the env?